TRANSFORMERS/SELECT_ARRAY/SELECT_ARRAY/SELECT_ARRAY.py

```diff
@@ -0,0 +1,35 @@
+from flojoy import flojoy, DataContainer
+
+@flojoy
+def SELECT_ARRAY(v, params):
+    '''
+    Node to convert an input array with multiple columns to the selected ordered pair.
+
+    For example, the SERIAL node can output x=time, y1=temperature, y2=pressure.
+    This node will select one of temperautre and pressure columns to output.
```

Comment:
temperature

Reply:
Done. Is there an extension or linter to check for typos in docstrings?